app.js

```diff
@@ -0,0 +1,77 @@
+
+console.log("Hello, World!"); //->English
+console.log("Silav, Cihan!");//->Kurdish
+console.log("Hoi, Wereld!"); //->Dutch
+
+console.log("I'm awesome!");
+
+let x =
+console.log('I think the value of x is bigger than 5');
+console.log("The value of x is 6");
+let x = 6;
+console.log(I think the value of x is 6);

```

Comment:
I think that this will will give an error message.


Did you mean to do this:

`console.log("I think the value of x is 6");`
?

Reply:
Changed to: console.log("the value of x: 6");